kura/org.eclipse.kura.wire.component.provider/OSGI-INF/metatype/org.eclipse.kura.wire.Timer.xml

```diff
@@ -28,8 +28,8 @@
             type="Integer"
             cardinality="0"
             required="true"
-            default="10"
-            description="Interval of the timer ticks, in seconds (Needs to be set if SIMPLE type is selected)"
+            default="10000"
```

Comment:
I think it would be better to provide 1 Attribute Definition for only taking integer and 1 Attribute Definition that lets user choose the Unit of Time for the provided number. We can provide all the units as present in `java.util.concurrent.TimeUnit`. This way it will be more flexible and customizable.